lib/cogctl/actions/users/util.ex

```diff
@@ -8,9 +8,16 @@ defmodule Cogctl.Actions.Users.Util do
 
   def render(user_info, group_info, role_info, sort) do
     Table.format(user_info, false)
-        <> "\n\nGroups:\n" <> Table.format(group_info, sort)
-        <> "\n\nRoles:\n" <> Table.format(role_info, sort)
+        <> "\n\nGroups:\n" <> render_groups(group_info, sort)
+        <> "\n\nRoles:\n" <> render_roles(role_info, sort)
```

Comment:
I agree with @vanstee. I like an optional flag like `--details` or `--groups`. But I think that might be for another PR.